fs/dafsserver/dafsserver.cpp

```diff
@@ -5834,41 +5834,43 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
 
     void checkTimeout()
     {
-        if (msTick()-clientcounttick>1000*60*60)
+        if (msTick() - clientcounttick > 1000 * 60 * 60)
         {
-            CriticalBlock block(ClientCountSect);
-            if (TF_TRACE_CLIENT_STATS && (ClientCount || MaxClientCount))
-                PROGLOG("Client count = %d, max = %d", ClientCount, MaxClientCount);
             clientcounttick = msTick();
```

Comment:
trivial: should have made observation before - would be more logical to reset this time at the end of the if block.